core/chains/evm/forwarders/forwarder_manager.go

```diff
@@ -0,0 +1,299 @@
+package forwarders
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/pkg/errors"
+	evmclient "github.com/smartcontractkit/chainlink/core/chains/evm/client"
+	evmlogpoller "github.com/smartcontractkit/chainlink/core/chains/evm/logpoller"
+	evmtypes "github.com/smartcontractkit/chainlink/core/chains/evm/types"
+	"github.com/smartcontractkit/chainlink/core/internal/gethwrappers/generated/authorized_forwarder"
+	"github.com/smartcontractkit/chainlink/core/internal/gethwrappers/generated/authorized_receiver"
+	"github.com/smartcontractkit/chainlink/core/internal/gethwrappers/generated/offchain_aggregator_wrapper"
+	"github.com/smartcontractkit/chainlink/core/logger"
+	"github.com/smartcontractkit/chainlink/core/utils"
+	"github.com/smartcontractkit/sqlx"
+)
+
+// Config encompasses config used by fwdmgr
+//go:generate mockery --recursive --name Config --output ./mocks/ --case=underscore --structname Config --filename config.go
+type Config interface {
+	LogSQL() bool
+}
+
+var AuthTopics = []common.Hash{
+	authorized_receiver.AuthorizedReceiverAuthorizedSendersChanged{}.Topic(),
+	offchain_aggregator_wrapper.OffchainAggregatorConfigSet{}.Topic(),
+}
+
+var ForwardABI = evmtypes.MustGetABI(authorized_forwarder.AuthorizedForwarderABI).Methods["forward"]
+
+var AuthLogCheckInterval = time.Duration(5 * time.Second)
+
+type FwdMgr struct {
+	ORM       ORM
+	config    Config
+	evmClient evmclient.Client
+	logger    logger.Logger
+	logpoller *evmlogpoller.LogPoller
+
+	// TODO(samhassan): sendersCache should be an LRU capped cache
+	sendersCache map[common.Address][]common.Address
+	latestBlock  int64
+
+	authRcvr    authorized_receiver.AuthorizedReceiverInterface
+	offchainAgg offchain_aggregator_wrapper.OffchainAggregatorInterface
+
+	ctx    context.Context
+	cancel context.CancelFunc
+
+	cacheMu sync.RWMutex
+	chStop  chan struct{}
+}
+
+func NewFwdMgr(db *sqlx.DB, cfg Config, client evmclient.Client, logpoller *evmlogpoller.LogPoller, lggr logger.Logger) *FwdMgr {
+	lggr = lggr.Named("FwdMgr")
+	lggr.Infow("Initializing EVM forwarder manager")
```

Comment:
This would be fine at debug level. Would also be good to include some basic info like the address of the forwarder contract if possible.

Reply:
It might be more interesting and aligned with other uses to move to `Start()` also :shrug:

Reply:
Done both, adding the address is suboptimal imho given that we will be tracking a number of them and they can be changed from CLI during run.